diff --git a/examples/server/CMakeLists.txt b/examples/server/CMakeLists.txt
index 859cd12..d53736e 100644
--- a/examples/server/CMakeLists.txt
+++ b/examples/server/CMakeLists.txt
@@ -5,6 +5,7 @@ add_executable(${TARGET} server.cpp json.hpp httplib.h)
 install(TARGETS ${TARGET} RUNTIME)
 target_compile_definitions(${TARGET} PRIVATE
     SERVER_VERBOSE=$<BOOL:${LLAMA_SERVER_VERBOSE}>
+    LOG_DEFAULT_FILE_NAME="" # disable log file.
 )
 target_link_libraries(${TARGET} PRIVATE common llama llava ${CMAKE_THREAD_LIBS_INIT})
 if (WIN32)
